h2o-algos/src/test/java/hex/deeplearning/DeepLearningCheckpointReporting.java

```diff
@@ -101,7 +98,7 @@
           durationAfter = durationAfter.substring(0, durationAfter.length()-4);
           double diff = Double.parseDouble(durationAfter) - Double.parseDouble(durationBefore);
           
-          Assert.assertTrue("Duration must be smooth; actual " + diff + ", expected at most " + sleepTime + " (before=" + durationBefore + ", after=" + durationAfter + ")", diff < sleepTime+1);
+          // Assert.assertTrue("Duration must be smooth; actual " + diff + ", expected at most " + sleepTime + " (before=" + durationBefore + ", after=" + durationAfter + ")", diff < sleepTime+1);
```

Comment:
maybe we can keep this assert at least for single node? might be not worth it but still worth considering

h2o-algos are test both 1-node and multi-node

Reply:
Good point.  Let me change it to reflect that.

Reply:
Let me change it and see if it fails any test and we can discuss then.